From 01eac3468b0c6ae3362d3c0521e57626e67b54cd Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Wed, 5 Jan 2022 19:46:31 +0200
Subject: [PATCH 52/89] mcm-imx8m-mini: remove the local rpmsg support

Remove the local rpmsg support, in order to comply with the following
commit:
aa414c3cbb0d5ee4610a20d3b79bed7a3f01c5ec:
LF-4134-2 arm64: dts: remove the local rpmsg support

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/mcm-imx8m-mini-m4.dts | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/arch/arm64/boot/dts/compulab/mcm-imx8m-mini-m4.dts b/arch/arm64/boot/dts/compulab/mcm-imx8m-mini-m4.dts
index 4b97ed29819a..2e9ce351bf95 100644
--- a/arch/arm64/boot/dts/compulab/mcm-imx8m-mini-m4.dts
+++ b/arch/arm64/boot/dts/compulab/mcm-imx8m-mini-m4.dts
@@ -16,16 +16,6 @@
 
 #include "mcm-imx8m-mini.dtsi"
 
-&rpmsg{
-	/*
-	 * 64K for one rpmsg instance:
-	 * --0xb8000000~0xb800ffff: pingpong
-	 */
-	vdev-nums = <1>;
-	reg = <0x0 0xb8000000 0x0 0x10000>;
-	status = "okay";
-};
-
 &uart1 {
 	status = "disabled";
 };
-- 
2.17.1

